Sources/Testing/Testing.docc/Attachments.md

```diff
@@ -25,8 +25,18 @@ protocol to create your own attachable types.
 - ``Attachable``
 - ``AttachableWrapper``
 
+
 <!-- TODO: set up DocC content for overlays if possible
 ### Attaching files to tests
 
-- ``/Foundation/URL/init(contentsOf:named:sourceLocation:)``
+- ``Attachment/init(contentsOf:named:sourceLocation:)``
+-->
+
+### Attaching images to tests
+
+- ``AttachableImageFormat``
+<!-- TODO: set up DocC content for overlays if possible
```

Comment:
Is it generating an error from DocC? Curious what the problem is

Reply:
Yes, it can't find the symbols because they're in other modules.